src/albums.go

```diff
@@ -117,3 +119,28 @@ func (a *Albums) PrintAlbumInfo() error {
 	}
 	return nil
 }
+
+// GetLstPhotosForWeek , Query the albums to retreive photo taken during the week of dataSelected
+func (a *Albums) GetLstPhotosForWeek(dateSelected time.Time) ([]*Photo, error) {
+	// retrieve information from the data provided
+	_, week := dateSelected.ISOWeek()
+
+	// variable to hold all picture
+	var photoOfTheWeeks []*Photo
+
+	// Go throw the album and print information for each pictures
+	for k := range a.lstPhotos {
+		for w := range a.lstPhotos[k] {
+			if w == week {
+				photoOfTheWeeks = append(photoOfTheWeeks, a.lstPhotos[k][w]...)
+			}
+		}
+	}
+
+	if len(photoOfTheWeeks) < 1 {
+		// Not sure for the error it's not really an issue. TODO review the error
+		return nil, fmt.Errorf("No pictures found for this year and week")
```

Comment:
as mentioned not for the error will see later, maybe for the web app, it will be nice to show a 404 ... :-/ 
Anyway, it will be easy to change later.